src/convertToObject.js

```diff
@@ -12,7 +12,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = sourceString.replace(/\n/g, '')
+    .split(';')
+    .map(element => element.trim())
+    .filter(element => element.length > 0);
+
+  const callback = (stylesResult, part) => {
```

Comment:
Краще конкретизувати, що саме робить колбек
```suggestion
  const convertToStylePairs = (stylesResult, part) => {
```